scripts/teamshow

```diff
@@ -44,12 +43,10 @@ class Teamshow(object):
 
     def get_portchannel_names(self):
         """
-            Get the portchannel names from minigraph.
+            Get the portchannel names from database.
         """
-        minigraph_path = '/etc/sonic/minigraph.xml'
-        machine_info = get_machine_info()
-        platform_info = get_platform_info(machine_info)
-        self.teams = parse_xml(minigraph_path, platform = platform_info)['PORTCHANNEL'].keys();
+        team_keys = self.db.keys(self.db.APPL_DB, PORT_CHANNEL_TABLE_PREFIX+"*")
+        self.teams = [key[len(PORT_CHANNEL_TABLE_PREFIX):] for key in team_keys]
```

Comment:
I think `team_keys` here should be read from CFG_DB instead of APPL_DB, as it should presents expected port channels instead of actual running ones. If a portchannel is defined in CFG_DB but not available in APPL_DB, there should be a line with status N/A.

Reply:
indeed, It is more reasonable to read from CONFIG_DB, thanks very much